run-checks

```diff
@@ -269,6 +269,10 @@ if [ "$UNIT" = 1 ]; then
             curl -s https://codecov.io/bash | bash /dev/stdin -f .coverage/coverage.out
         fi
     fi
+
+    # python unit test for mountinfo-tool
+    command -v python2 && python2 ./tests/lib/bin/mountinfo-tool --self-test
```

Comment:
With the tool name I suppose we cannot really make it work with `python3 -m unittest <path>`.

Reply:
I tried to make it a standalone tool that doesn't involve the python module system at all, apart from the standard library.